packages/ui/src/components/style/core/global.less

```diff
@@ -6,4 +6,16 @@
 :root {
   --condo-font-fallback: sans-serif;
   --condo-font-fallback-mono: consolas, monaco, "Andale Mono", "Ubuntu Mono", monospace;
-}
\ No newline at end of file
+}
+
+@from-mobile-small: ~"screen and (min-width: 0px)";
+@from-mobile-large: ~"screen and (min-width: 360px)";
```

Comment:
360px is intersecting, which can cause some bugs. Use semi-intervals:

[a; b) .... [b; c) .... [c ; d) .... [ d; +inf)

So, in particular case:
[0; 359] ... [360; 479] and so on

Reply:
done